tests/memcpy/1-src.ll

```diff
@@ -0,0 +1,12 @@
+declare void @llvm.memcpy.p0i8.p0i8.i32(i8*, i8*, i32, i1)
+
+define i8 @f() {
+  %p1 = alloca [16 x i8]
```

Comment:
Have you run these tests with e.g. llvm-tv?  I was fairly sure that arrays aren't implemented yet

Reply:
I ran the tests using alive-tv and their results are as expected.

I think that Alive2 encodes the code to alloca n bytes.

For example, 'alloca [16 x i8]' is encodded 'alloca 16 bytes'.
And 'alloca i2' is encode 'alloca 1 bytes'.

Reply:
ok, we (well, I) need to extend the unit test runner to run alive-tv. I'll do that later this week.